frontend/src/modules/navigation/components/Header.jsx

```diff
@@ -31,7 +31,11 @@ const TappNavItems = () => (
     </Nav>
 );
 
-const CpNavItems = () => <Nav className="mr-auto" />;
+const CpNavItems = () => (
```

Comment:
We have `CP` and `Control Panel`. This is a really confusing naming convention. If it's not exported, don't give it an overly specific name.

Reply:
Would it make sense if I changed `Control Panel` to `Offers` since its main purpose is to handle creation/management of offers?

Reply:
Remove the `Cp` from `CpNavItems`